src/loaders/ObjectLoader.js

```diff
@@ -632,11 +679,73 @@ Object.assign( ObjectLoader.prototype, {
 			if ( data.visible !== undefined ) object.visible = data.visible;
 			if ( data.userData !== undefined ) object.userData = data.userData;
 
-			if ( data.children !== undefined ) {
+			/*
+			 * SkinnedMesh creates Bone instances as its children in the constructor
+			 * so skip Bone instance creation here.
+			 */
+			if ( data.type === 'SkinnedMesh' ) {
+
+				var scope = this;
+
+				function traverse ( data ) {
+
+					if ( data.children !== undefined ) {
 
-				for ( var child in data.children ) {
+						for ( var child in data.children ) {
 
-					object.add( this.parseObject( data.children[ child ], geometries, materials ) );
+							if ( data.children[ child ].type !== 'Bone' ) {
+
+								var parent;
+
+								if( data.type === 'SkinnedMesh' ) {
+
+									parent = object;
+
+								} else if ( data.type === 'Bone' ) {
+
+									// should make a method THREE.Skeleton.findBoneByName() ?
+									for ( var i = 0, il = object.skeleton.bones.length; i < il; i ++ ) {
+
+										if ( object.skeleton.bones[ i ].name === data.name ) {
+
+											parent = object.skeleton.bones[ i ];
+											break;
+
+										}
+
+									}
+
+								}
+
+								if ( parent ) {
+
+									parent.add( scope.parseObject( data.children[ child ], geometries, materials, skeletons ) );
+
+								}
+
+							} else {
+
+								traverse( data.children[ child ] );
+
+							}
+
+						}
+
+					}
+
+				}
+
+				traverse( data );
+
+			} else {
```

Comment:
So this seems to be the last thing to fix...

Maybe we should make it so `Bone`s can't be part of the scene graph? I think the main benefit of this was so we could add object as children of bones. Also, it allows moving the bones in the editor right now, but I think we could re-implement those things in a different way.

But the good thing is that It would simplify/solve this, right?